### PR TITLE
Update the 2.8.0.1 download link for hub sync

### PR DESCRIPTION
**Pull request by BewareMyPower**

### CODE REVIEW

**Comment by BewareMyPower** [comment]
After it's merged, I'll cherry-pick it to branch-2.8.0

**Review by dockerzhang — APPROVED** [review]

File: docs/kop.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-dockerfile: "https://github.com/streamnative/kop/releases/download/v2.8.0-rc-202105291235//pulsar-protocol-handler-kafka-2.8.0-rc-202105291235.nar"
+dockerfile: "https://github.com/streamnative/kop/releases/download/v2.8.0.1/pulsar-protocol-handler-kafka-2.8.0.1.nar"
```
Comment (Jennifer88huang) [inline docs/kop.md:2]:
```suggestion
download: "https://github.com/streamnative/kop/releases/download/v2.8.0.1/pulsar-protocol-handler-kafka-2.8.0.1.nar"
```

Comment (Jennifer88huang) [reply docs/kop.md:2]: @tuteng since we don't have docker image for KoP, shall we replace it with "download"?

**Comment by BewareMyPower** [comment]
@Jennifer88huang PTAL again

**Review by Jennifer88huang — APPROVED** [review]

**Pull request merged**